cider/tests/nextgen/jitlib/JITLibTest.cpp

```diff
@@ -141,6 +174,112 @@ TEST_F(JITLibTests, LogicalOpTest) {
       true, false, false, [](JITValue& a, JITValue& b) { return !a; });
   executeBinaryOp<JITTypeTag::BOOL>(
       false, true, true, [](JITValue& a, JITValue& b) { return !a; });
+
+  // and
+  executeBinaryOp<JITTypeTag::BOOL>(
+      true, false, false, [](JITValue& a, JITValue& b) { return a && b && true; });
+  executeBinaryOp<JITTypeTag::BOOL>(
+      true, true, true, [](JITValue& a, JITValue& b) { return a && b && true; });
+  executeBinaryOp<JITTypeTag::BOOL>(
+      true, false, false, [](JITValue& a, JITValue& b) { return a && b && false; });
+  executeBinaryOp<JITTypeTag::BOOL>(
+      false, false, false, [](JITValue& a, JITValue& b) { return a && b && true; });
```

Comment:
Change these cases as ```a && b && true```, ```a && true && b```, ```false && a && b``` to test the combination correctness.

Reply:
added